content/docs/command-reference/import-url.md

```diff
@@ -186,14 +195,11 @@ Let's take a look at the changes to the `data.xml.dvc`:
 The `etag` field in the `.dvc` file contains the
 [ETag](https://en.wikipedia.org/wiki/HTTP_ETag) recorded from the HTTP request.
 If the remote file changes, its ETag will be different. This metadata allows DVC
-to determine whether its necessary to download it again.
+to determine whether it's necessary to download it again.
 
 > See [`.dvc` files](/doc/user-guide/dvc-files-and-directories#dvc-files) for
 > more details on the format above.
 
-You may want to get out of and remove the `example-get-started/` directory after
-trying this example (especially if trying out the following one).
-
```

Comment:
I believe this statement is irrelevant here since anyone could arrive at this page without going through the `User Guide/Getting-Started` and it does not hold a direct info about the `import-url` command. It seems like the content was borrowed from one of those pages.

Reply:
I don't think you followed the entire examples section first. That would need to be done to determine this. Feel free to do so and submit a different PR if you find this unnecessary, but it's out of scope for this PR. Please roll back here. Thanks

Reply:
Alright.
But do you think it should be removed? 
> I don't think you followed the entire examples section first.  

Actually, since this is in `command reference`, I thought it would be good to keep this file independent of the `Getting Started` module

Reply:
I don't think it should be removed. But like I mentioned, I would need to follow the entire examples section first to decide.

> would be good to keep this file independent of the Getting Started

It is. Again, I don't think you read the whole examples section 🙂 

Please roll back.

Reply:
Oh okay. Now I get it. It's just related to the example above.
My apologies :smile: